cmd/clair-load-test/reports.go

```diff
@@ -132,20 +133,24 @@ func NewConfig(c *cli.Context) *TestConfig {
 	}
 }
 
+// fetchLayers calculates the layers number used to fetch images.
+// It returns an integer indicating amount of layers.
+func fetchLayers(ctx context.Context, layers int, validLayers []int) int {
+	rand.Seed(time.Now().UnixNano())
```

Comment:
Nit: but maybe consider requiring go1.20 (or 1.21) and avoiding having to do this?

Reply:
Upgraded to go 1.21 and removed this statement.